packages/components/src/templates/next/components/internal/SearchableTable/CellContent.tsx

```diff
@@ -0,0 +1,37 @@
+import type { IsomerSitemap, LinkComponentType } from "~/types"
+import { getReferenceLinkHref } from "~/utils"
+import BaseParagraph from "../BaseParagraph"
+
+const HYPERLINK_EXCEL_FUNCTION = "=HYPERLINK("
+
+interface CellContentProps {
+  content: string | number
+  sitemap: IsomerSitemap
+  LinkComponent?: LinkComponentType
+}
+
+export const CellContent = ({
+  content,
+  sitemap,
+  LinkComponent,
+}: CellContentProps) => {
+  if (
+    typeof content === "string" &&
+    content.startsWith(HYPERLINK_EXCEL_FUNCTION) &&
+    content.endsWith(")")
+  ) {
+    const link = content.slice(HYPERLINK_EXCEL_FUNCTION.length, -1)
+    const [linkHref, linkText] = link.split(",")
```

Comment:
this is abit confusing to me - i'm not really clear what `link` here refers to! on initial glance, i assumed that this wuold be `www.isomer.gov.sg` (for example), so i didn't quite understand how the `,` got in there

Reply:
This is an Excel-style of hyperlinks: https://support.microsoft.com/en-us/office/hyperlink-function-333c7ce6-c5ae-4164-9c47-7de9b76f577f

So to add a link, the cell contents will be like `=HYPERLINK(https://www.isomer.gov.sg,Isomer)`, and it will be rendered as [Isomer](https://www.isomer.gov.sg).